apis/v1alpha1/gateway_types.go

```diff
@@ -377,9 +372,8 @@ type RouteBindingSelector struct {
 	//
 	// Support: Core
 	//
-	// +optional
 	// +kubebuilder:default={onlySameNamespace:true}
-	RouteNamespaces RouteNamespaces `json:"routeNamespaces,omitempty"`
+	RouteNamespaces *RouteNamespaces `json:"routeNamespaces,omitempty"`
```

Comment:
`RouteNamespaces` should continue to be a value.

Reply:
Updated, thanks!